packages/vats/src/types.d.ts

```diff
@@ -264,3 +265,24 @@ type SendPacketDownCall = {
   packet: IBCPacket;
   relativeTimeoutNs: bigint;
 };
+
+export type VTransferIBCEvent = {
```

Comment:
docstring plz

for [product API reference docs](https://agoric-sdk.pages.dev/)